### ISSUE TITLE
It is not possible to create an Ethernet network with MaximumBandwith greater then 20Gb/s

### ISSUE THREAD

**Issue by jacek-skwara** [HewlettPackard/POSH-HPOneView#442]
### Expected Behavior
New-HPOVNetwork commandlet allows to create an Ethernet network with MaximumBandwith greater then 20GB/s

### Actual Behavior
The commandlet seems to have a fixed limit of MaximumBandwith set to 20Gb/s

### Steps to reproduce
```powershell
New-HPOVNetwork -Name VLAN_test -type Ethernet -VLANType Tagged -VlanId 2000 -TypicalBandwidth 2500 -MaximumBandwidth 25000 -verbose

New-HPOVNetwork : Cannot validate argument on parameter 'MaximumBandwidth'. The 25000 argument is greater than the maximum allowed range of 20000. Supply an argument that is less than or equal to 20000 and then try the command again.
At line:1 char:119
+ ... d -VlanId 2000 -TypicalBandwidth 2500 -MaximumBandwidth 25000 -verbos ...
+                                                             ~~~~~
    + CategoryInfo          : InvalidData: (:) [New-HPOVNetwork], ParameterBindingValidationException
    + FullyQualifiedErrorId : ParameterArgumentValidationError,New-HPOVNetwork
```
### Version Information
HPE OneView PowerShell Library Version (`Get-HPOVVersion` or `$PSLibraryVersion`):
HPE OneView Appliance Version (`Get-HPOVVersion -ApplianceVer`):  

```powershell
Get-HPOVVersion -ApplianceVer

xxxxxxxxxxxxxxxxxxxxxx           LibraryVersion Path
-------------------------           -------------- ----
ApplianceVersion: 5.00.01.410269.00 5.0.2226.2609  C:\Program Files\WindowsPowerShell\Modules\HPOneView.500\5.0.2226.2609
```

Output from `$PSVersionTable` on your Windows Host:

```PowerShell
$PSVersionTable

Name                           Value
----                           -----
PSVersion                      5.1.17763.771
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.17763.771
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1

```

**Comment by ChrisLynchHPE** [comment]
Thanks for reporting this.  The typical and maximum bandwidth should be allowed up to 50000 (50Gbps).  Also, Set-HPOVNetwork and New-HPOVNetworkSet need to be updated.  I'm pushing out a few updates today, and this will be part of the next 5.00 update.

**Comment by ChrisLynchHPE** [comment]
This is fixed in [Release 5.0.2295.3359](https://github.com/HewlettPackard/POSH-HPOneView/releases/tag/v5.0.2295.3359) and in [PowerShell Gallery](https://www.powershellgallery.com/packages/HPOneView.500/5.0.2295.3359).

Please verify before this issue is closed.

**Comment by jacek-skwara** [comment]
I have just verified the new module at the customer site and the issue is gone. Thanks a lot for taking care. The *HPOVNetwork* commandlets are very important so they can be used in Synergy scripting now.
